template.rb

```diff
@@ -109,6 +118,8 @@ def print_announcements
 run_db_setup = yes?("Run db setup steps? (y/n)")
 
 ## Start of app customizations
+insert_into_file "Gemfile", "\n#{gem_ruby_entry}\n", after: /^source "https.*\n/
```

Comment:
@zjrgov this will fix the deploy issue of cloud.gov not knowing what version of ruby to use